### ISSUE TITLE
Not all languages available in spell checker

### ISSUE THREAD

**Issue by laszlovl** [GrapheneOS/os-issue-tracker#358]
Spun off from #247

The keyboard/spell checker currently only supports the following languages: English, French, German, Italian, Spanish, Russian and Portuguese

However, looking at https://github.com/GrapheneOS/platform_packages_inputmethods_LatinIME/tree/11/dictionaries it seems a lot more languages have a dictionary available. It would be great if these additional languages were also made available as spell check option.

**Comment by jorisvandijk** [comment]
Want to bump this issue as I also very much miss spelling corrections (in Dutch). I seem to be a very messy typist.

**Comment by thestinger** [comment]
Please stick to using the thumbs up reaction to express support for a change. Comments send an email notification to everyone following the issue and as the owner of the repository, I'm following every issue on it by default.

**Comment by adamc199** [comment]
Another fluent Dutch speaking American here who also would love to see the Dutch language added to spell check.

**Comment by thestinger** [comment]
> Please stick to using the thumbs up reaction to express support for a change. Comments send an email notification to everyone following the issue and as the owner of the repository, I'm following every issue on it by default.

**Comment by thestinger** [comment]
You're welcome to use a different keyboard or to contribute.

**Comment by adamc199** [comment]
> You're welcome to use a different keyboard or to contribute.

I installed the Dutch keyboard to no avail. 
Happy to contribute. Does this mean an entire dictionary must be added the the project?

**Comment by IT-Exilion** [comment]
Added pull request for SK(slovak) language

**Comment by thestinger** [comment]
Part of implementing this is determining what has to be done. Changes were submitted to add Czech and Dutch spellchecking support. It appears that this may not be working. There won't be improvements without contributors who use those languages contributing including making sure the changes work.

**Comment by thestinger** [comment]
Contributions that are properly tested are welcome. This issue serves no purpose since there's no definition of what is meant to be done to complete it. There is no plan to specifically add support for those languages.

**Comment by dewi-g** [comment]
I'm Czech and I can confirm the Czech suggestions/autocomplete don't work as well. The keyboard layout works fine with all our weird shit above letters (ščěřžť) but no suggestions apart from contact names and custom words work. I don't know what can be done to fix it but I'm willing to help if someone points me in some direction.
In the meantime, I came up with an idea to add complete czech dictionary to the custom dictionary, which wouldn't be as good as next word suggestions but better than nothing, and I suppose the keyboard would learn overtime. However I did not find any way to add more words at once so that doesn't seem as an option for now. I also tried many other keyboards but none of them has czech support (yet, the FlorisBoard seems to try to implement suggestions soon).
So I wanted to ask if there is a way to add whole dictionary into personal dictionary at once and also if there is any concrete step I can take to help or if you have some idea what I could poke into to try to help. Thanks (and I hope this is the right place to ask)
